i18n/ja/docusaurus-plugin-content-docs/current/guides/mobile-plugin-ima-android.md

```diff
@@ -0,0 +1,63 @@
+---
+title: UID2 IMA Plugin for Android
+description: A guide for publishers using GIA for ad requests on Android apps.
+hide_table_of_contents: false
+sidebar_position: 14
+---
+
+# UID2 IMA Plugin for Android Integration Guide
+
+The UID2 Interactive Media Ads (IMA) Plugin for Android enables publishers that use the [Google Interactive Media Ads SDK](https://developers.google.com/interactive-media-ads/docs/sdks/html5/client-side/) to send [UID2 tokens](../ref-info/glossary-uid.md#gl-uid2-token) as [secure signals](https://support.google.com/admob/answer/11556288?hl=en-GB) in ad requests from Android apps. This is done automatically, with no direct coding needed from the app developer.
```

Comment:
The IMA SDK link is also incorrect.  It should be --> 

https://developers.google.com/interactive-media-ads/docs/sdks/android/client-side

Reply:
Fixed, see update, thx.

Reply:
@bleege I changed Line 32 to match. It seemed a better choice. Over to you. :-)